src/ostorlab/serve_app/oxo.py

```diff
@@ -266,6 +269,96 @@ def mutate(
             return DeleteScanMutation(result=True)
 
 
+class CreateAssetsMutation(graphene.Mutation):
+    """Create asset mutation."""
+
+    class Arguments:
+        assets = graphene.List(types.AssetInputType, required=True)
+
+    assets = graphene.List(types.AssetType)
+
+    @staticmethod
+    def mutate(
+        root, info: graphql_base.ResolveInfo, assets: List[types.AssetInputType]
+    ):
+        """Create asset mutation."""
+        created_assets = []
+        errors = []
+        config_manager = configuration_manager.ConfigurationManager()
+        for asset in assets:
+            error_message = CreateAssetsMutation._validate(asset)
+            if error_message is not None:
+                errors.append(error_message)
+                continue
```

Comment:
Shouldn't we validate all the received assets first, and if only one issue is found, we raise an error? because there is no need to process other valid assets if we raise an error at the end and lose all the IDs of the created ones.